app/bundles/EmailBundle/Assets/js/email.js

```diff
@@ -236,11 +236,13 @@ Mautic.selectEmailType = function(emailType) {
         mQuery('#segmentTranslationParent').removeClass('hide');
         mQuery('#templateTranslationParent').addClass('hide');
         mQuery('.page-header h3').text(mauticLang.newListEmail);
+        mQuery('#scheduleOptions').removeClass('hide');
```

Comment:
Why to hide it for both list and template types? It's the all types there are. From the programming perspective, it shouldn't be in the if nor else blocks but bellow it as you want to apply it to both. But from the product perspective, why do we have a feature that we want to hide for all options?

Reply:
Oops, this is `removeClass`. Makes sense then.